### PR TITLE
New style of cards on all categories list

### PR DESCRIPTION
**Pull request by kgogola**
https://tree.taiga.io/project/fierycod-knit-front-vuejs/task/177

Continuation of previous PR : https://github.com/knit-pk/homepage-nuxtjs/pull/135

### CODE REVIEW

File: components/article/ArticleCardList.vue [hunk 1]
```diff
@@ -174,10 +174,44 @@ export default {
   /* stylelint-disable */
   &--horizontal {
     .article-card {
+    @media (min-width: 1060px) {
       &:nth-of-type(n+3) {
         flex-basis: 100%;
+        display: block;
+        min-height: 0;
+        .article-card__thumbnail-link {
```
Comment (FieryCod) [inline components/article/ArticleCardList.vue]: Please add the newline after each `{}` block

File: components/article/ArticleCardList.vue [hunk 2]
```diff
@@ -174,10 +174,44 @@ export default {
...
+          margin-top: 15px;
+        }
+        .article-card__title-link {
+          order: 1;
+          margin-left: -10px;
+        }
+        .article-card__tags-wrapper {
+          order: 2;
+        }
+        .article-card__single-tag {
```
Comment (FieryCod) [inline components/article/ArticleCardList.vue]: Please do not change the size of a tag.

File: components/article/ArticleCardList.vue [hunk 3]
```diff
@@ -174,9 +174,57 @@ export default {
   /* stylelint-disable */
   &--horizontal {
     .article-card {
+      
+      // height: 360px;
```
Comment (FieryCod) [inline components/article/ArticleCardList.vue]: Could you please remove that comment?

**Comment by FieryCod** [comment]
Please fix issues

File: components/article/ArticleCardList.vue [hunk 4]
```diff
@@ -174,9 +174,53 @@ export default {
   /* stylelint-disable */
   &--horizontal {
     .article-card {
+    @media (min-width: 1060px) {
```
Comment (FieryCod) [inline components/article/ArticleCardList.vue]: This block should be on next line and have extra 2 spaces indent.

File: components/article/ArticleCardList.vue [hunk 5]
```diff
@@ -174,9 +174,53 @@ export default {
...
       &:nth-of-type(n+3) {
         flex-basis: 100%;
+        display: block;
+        min-height: 20px;
+        height: 264px;
+        .article-card__thumbnail-link {
```
Comment (FieryCod) [inline components/article/ArticleCardList.vue]: Extra newline above that line please

File: components/article/ArticleCardList.vue [hunk 6]
```diff
@@ -174,9 +174,53 @@ export default {
...
+          width: 100%;
+          justify-content: flex-end;
+          height: 64px;
+        }
+
+        .article-card__stats {
+          margin-top: -20px;
+        }
       }
+}
```
Comment (FieryCod) [inline components/article/ArticleCardList.vue]: wrong indent

**Comment by FieryCod** [comment]
One more and we are ready to merge :)

**Comment by kgogola** [comment]
should I delete previous commits ?

**Comment by FieryCod** [comment]
You should squash them as I shown last weekend. Please squash them to one commit and we are ready to go!

**Pull request merged**